data/autoyast_sle15/autoyast_scc_up.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <bootloader>
+    <global>
+      <activate>false</activate>
+      <boot_extended>false</boot_extended>
+      <boot_mbr>true</boot_mbr>
+      <boot_root>true</boot_root>
+      <generic_mbr>false</generic_mbr>
+      <timeout config:type="integer">5</timeout>
+    </global>
+    <loader_type>grub2</loader_type>
+  </bootloader>
+  <general>
+    <mode>
+      <confirm config:type="boolean">false</confirm>
+      <second_stage config:type="boolean">false</second_stage>
+    </mode>
+    <signature-handling>
+      <accept_file_without_checksum config:type="boolean">true</accept_file_without_checksum>
+      <accept_non_trusted_gpg_key config:type="boolean">true</accept_non_trusted_gpg_key>
+      <accept_unknown_gpg_key config:type="boolean">true</accept_unknown_gpg_key>
+      <accept_unsigned_file config:type="boolean">false</accept_unsigned_file>
+      <accept_verification_failed config:type="boolean">false</accept_verification_failed>
+      <import_gpg_key config:type="boolean">true</import_gpg_key>
+    </signature-handling>
+  </general>
+  <keyboard>
+    <keymap>english-us</keymap>
+  </keyboard>
+  <language>
+    <language>en_US</language>
+    <languages>en_US</languages>
+  </language>
+  <report>
+    <errors>
+      <show config:type="boolean">true</show>
+      <log config:type="boolean">true</log>
+      <timeout config:type="integer">0</timeout>
+    </errors>
+    <warnings>
+      <show config:type="boolean">true</show>
+      <log config:type="boolean">true</log>
+      <timeout config:type="integer">0</timeout>
+    </warnings>
+    <messages>
+      <show config:type="boolean">true</show>
+      <log config:type="boolean">true</log>
+      <timeout config:type="integer">0</timeout>
+    </messages>
+    <yesno_messages>
+      <show config:type="boolean">true</show>
+      <log config:type="boolean">true</log>
+      <timeout config:type="integer">0</timeout>
+    </yesno_messages>
+  </report>
+  <software>
+    <products config:type="list">
+        <product>SLES</product>
+    </products>
+  </software>
+  <upgrade>
+    <stop_on_solver_conflict config:type="boolean">true</stop_on_solver_conflict>
+  </upgrade>
+</profile>
```

Comment:
The profile looks not properly for sle15, that maybe the reason of failure: http://openqa-apac1.suse.de/tests/555#step/installation/44
Please refer this: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/data/autoyast_sle15/autoyast_gnome.xml

Reply:
Or maybe you need this: https://github.com/yast/yast-registration/wiki/Available-SCC-Extensions-for-Use-in-Autoyast#sles-15
@rwx788 as a autoyast expert, could you give some hint about autoyast profile on sle15 upgrade? Thanks.

Reply:
I actually was surprised that no repos are defined and no modules activated in the profile, but I'm not sure how is it supposed to look like

Reply:
I'm not sure about upgrade scenarios, I guess it's better to ask on #yast irc channel. But I would expect registration part similarly as we have [here](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/data/autoyast_sle15/autoyast_btrfs.xml#L4), for instance.

Reply:
@mitiao , @rwx788 
Yes, there is no registration code and modules in the autoyast profile, because it showed an error when I included them in the profile, such as:
  <suse_register>
    <do_registration config:type="boolean">true</do_registration>
    <email/>
    <reg_code>{{SCC_REGCODE}}</reg_code>
    <install_updates config:type="boolean">true</install_updates>
    <reg_server>{{SCC_URL}}</reg_server>
    <addons config:type="list">
      <addon>
        <name>sle-module-server-applications</name>
        <version>{{VERSION}}</version>
        <arch>{{ARCH}}</arch>
      </addon>
      <addon>
        <name>sle-module-desktop-applications</name>
        <version>{{VERSION}}</version>
        <arch>{{ARCH}}</arch>
      </addon>
      <addon>
        <name>sle-legacy</name>
        <version>{{VERSION}}</version>
        <arch>{{ARCH}}</arch>
      </addon>
    </addons>
  </suse_register>
![screenshot_autoupgrade_scc](https://user-images.githubusercontent.com/16044897/37954720-3d94d072-31d9-11e8-9646-58e71c556be9.png)

Maybe some section is still missing in my autoyast profile, but I can't figure it out.

Reply:
Do you pass settings? We expand variables in {{}} which is done in prepare_profile test module. It's scheduled with AUTOYAST_PREPARE_PROFILE setting. Not sure if smt registration has any special flow. As I've said, best way to ask yast team, which I can do later today. Unfortunately, we have end of the sprint and some meetings.